### DISCUSSION TITLE
Setting weights for Longformer classification model

### DISCUSSION THREAD

**Discussion by Timliuzhaolu** [ThilinaRajapakse/simpletransformers#947] [Q&A]
HI

I was trying to set class weights in a classification model. However I can only do so using the sample code which uses roberta. When I try to implement the same code using my fine-tuned longformer it says TypeError: __init__() got an unexpected keyword argument 'weight'. Is it because long former does not support this weight setting feature or am I doing something wrong?
![image](https://user-images.githubusercontent.com/74063834/104313055-62f32f00-54cf-11eb-8883-5a8bb8ed59f9.png)

**Comment by ThilinaRajapakse** [comment]
You aren't doing anything wrong. Longformer models don't currently support the weights feature.
